hq/app/aws/iam/CredentialsReport.scala

```diff
@@ -0,0 +1,81 @@
+package aws.iam
+
+import java.io.StringReader
+
+import model.{IAMCredential, IAMCredentialsReport}
+import com.github.tototoshi.csv._
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
+import com.amazonaws.regions.{Region, Regions}
+import com.amazonaws.services.identitymanagement.model.{GenerateCredentialReportResult, GetCredentialReportResult}
+
+
+object CredentialsReport {
+
+  def isComplete(report: GenerateCredentialReportResult) = report.getState == "COMPLETE"
+
+  def extractReport(result: GetCredentialReportResult): IAMCredentialsReport = {
+    val content = result.getContent
+    val dest = new Array[Byte](content.capacity())
+    content.get(dest, 0, dest.length)
+      IAMCredentialsReport(
+        new DateTime(result.getGeneratedTime),
+        parseCredentialsReport(new String(dest))
+      )
+  }
+
+  // parsing the credentials report CSV file
+  val isoDateTimeParser = ISODateTimeFormat.dateTimeParser().withZoneUTC()
```

Comment:
I think this should be in [DateUtils](https://github.com/guardian/security-hq/blob/master/hq/app/logic/DateUtils.scala)